pyface/tests/test_about_dialog.py

```diff
@@ -8,16 +8,26 @@
 from ..toolkit import toolkit_object
 from ..window import Window
 
+GuiTestAssistant = toolkit_object('util.gui_test_assistant:GuiTestAssistant')
+no_gui_test_assistant = (GuiTestAssistant.__name__ == 'Unimplemented')
+
 ModalDialogTester = toolkit_object('util.modal_dialog_tester:ModalDialogTester')
 no_modal_dialog_tester = (ModalDialogTester.__name__ == 'Unimplemented')
 
 
-class TestAboutDialog(unittest.TestCase):
+@unittest.skipIf(no_gui_test_assistant, 'No GuiTestAssistant')
+class TestAboutDialog(unittest.TestCase, GuiTestAssistant):
 
     def setUp(self):
-        self.gui = GUI()
+        GuiTestAssistant.setUp(self)
```

Comment:
Can `super` be used if `GUITestAssistant` was the first base class? I think mixin classes in Python have to be first base class to play nice with inheritance and super